pages/Getting Started/Installation.md

```diff
@@ -251,7 +248,7 @@ zypper in gcc-c++ git meson cmake "pkgconfig(cairo)" "pkgconfig(egl)" "pkgconfig
 
 ```plain
 pkg install git pkgconf gmake gcc evdev-proto cmake wayland-protocols wayland libglvnd libxkbcommon libinput cairo pango pixman libxcb
-pkg install meson jq `pkg rquery %dn wlroots` hwdata libdisplay-info libliftoff
+pkg install meson jq hwdata libdisplay-info libliftoff
```

Comment:
Isn't aquamarine also needed? Or is it pulled in by the `hyprland` package/port.

Reply:
dossnt have a pkg yet